plugins/lightspeed-backend/README.md

```diff
@@ -0,0 +1,78 @@
+# Lightspeed Backend
+
+This is the lightspeed backend plugin that enables you to interact with any LLM server running a model with OpenAI's API compatibility.
```

Comment:
New plugins should be contributed to https://github.com/backstage/community-plugins  as this repo is deprecated.